cmd/caboose/main.go

```diff
@@ -84,5 +85,6 @@ func main1() int {
 		log.Println(err)
 		return 1
 	}
+	time.Sleep(5 * time.Second)
```

Comment:
why do we need this?

Reply:
This is to wait for some time to ensure we actually see a 200 from the log server as a log line. I'll remove it.

Reply:
Removed.

Reply:
@aarshkshah1992  but.. did you push the removal? :upside_down_face:  I still see it :D